test/cpp/api/init.cpp

```diff
@@ -0,0 +1,88 @@
+#include <gtest/gtest.h>
+
+#include <torch/nn/init.h>
+#include <torch/nn/modules/linear.h>
+
+#include <test/cpp/api/init_baseline.h>
+#include <test/cpp/api/support.h>
+
+#include <vector>
+
+void check_exact_values(
+    std::vector<torch::Tensor> parameters,
```

Comment:
We don't care about perf here but `const&` if you want to avoid copying the whole vector